CONTRIBUTING.md

```diff
@@ -88,3 +88,11 @@ GOOD: * [Another Awesome Book](http://example.com/book.pdf) (PDF)
 BAD : * [Another Awesome Book](http://example.com/book.pdf)- John Doe
 GOOD: * [Another Awesome Book](http://example.com/book.pdf) - John Doe (PDF)
 ```
+
+- Multiple formats:
+
+```
+BAD : * [Another Awesome Book](http://example.com/book.html)- John Doe (HTML)
+BAD : * [Another Awesome Book](https://downloads.example.org/book.html)- John Doe (download site)
+GOOD: * [Another Awesome Book](http://example.com/book.pdf) - John Doe (HTML) [(PDF, EPUB)](https://downloads.example.org/book.html)
```

Comment:
Looks like `http://example.com/book.pdf` points to a PDF but is noted as `HTML`. Could you please use `http://example.com/book/` as target for the first like of this line?

Thanks!